presto-hive-hadoop2/pom.xml

```diff
@@ -98,19 +98,24 @@
         </dependency>
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <excludedGroups>hive,hive-s3</excludedGroups>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
     <profiles>
```

Comment:
You could create a README.md file in this module which would explain how to run this

Reply:
unlikely to be found ;)